plugins/inputs/tacacs/sample.conf

```diff
@@ -0,0 +1,14 @@
+[[inputs.tacacs]]
```

Comment:
Is this a new requirement since some months ago?  Thought its enough to have it in README.md (its also not present in the radius plugin in sample.conf)